README.md

```
@@ -0,0 +1,303 @@
+
+### Get the code
+
+```bash
+git clone https://github.com/jayvicsanantonio/VaderCleaner.git
+cd VaderCleaner
+open VaderCleaner.xcodeproj
+```
+
+### Project layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)